src/main/java/uk/gov/justice/digital/delius/controller/StaffController.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.justice.digital.delius.controller;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.HttpServerErrorException;
+import uk.gov.justice.digital.delius.data.api.AccessLimitation;
+import uk.gov.justice.digital.delius.data.api.ManagedOffender;
+import uk.gov.justice.digital.delius.jwt.Jwt;
+import uk.gov.justice.digital.delius.jwt.JwtValidation;
+import uk.gov.justice.digital.delius.service.NoSuchUserException;
+import uk.gov.justice.digital.delius.service.StaffService;
+
+import java.util.List;
+
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.OK;
+
+@RestController
+@Slf4j
+@Api(tags = "Staff")
+public class StaffController
+{
+    private StaffService staffService;
+    private Jwt jwt;
```

Comment:
I don't think this is needed

Reply:
You are right. I think this has been added, then functionality removed or moved elsewhere and its no longer used. I will remove it.